<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VtaSetting</class>
 <widget class="QDialog" name="VtaSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>552</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vta Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="LblSettingOutPut">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Vta-By Deep Govind,   Email:deepgovind4@gmail.com,  Mob No. : 7388966141</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="VtaSet">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Vta Add Voice</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Write Voice Text :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TxtVtaVoice"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QComboBox" name="CommandType">
               <item>
                <property name="text">
                 <string>ToCMD</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>DoubleClick</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SingleClick</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SendKey</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Select Type</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Write Arguments :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="CBKey">
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="CBKeyChar">
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TxtVtaCommand"/>
             </item>
             <item>
              <widget class="QPushButton" name="BtnSetImage">
               <property name="text">
                <string>Capture Image</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="BtnExportCommand">
             <property name="text">
              <string>Export Command</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnImportCommand">
             <property name="text">
              <string>Import Command</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="BtnAdd">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QTableWidget" name="VtaVoiceTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::IgnoreAction</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="columnCount">
            <number>4</number>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Vta Default Voice</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QCheckBox" name="CHRecog">
                <property name="text">
                 <string>off Recognize</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Start English Typing Txt : </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="TxtEnglishTyping"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Start Hindi Typing Txt   :   </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="TxtHindiTyping"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Start Hindi Dictionary :     </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="TxtHindiDictionary"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QCheckBox" name="CHChatRecog">
                   <property name="text">
                    <string>off Chat Recognize</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Set Time To Sleep :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="VtaSleepAfter">
                 <property name="text">
                  <string>5</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Second</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Set Txt To Get Up :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="VtaGetUpTxt">
                 <property name="text">
                  <string>get up</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Set Txt To Sleep   :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="VtaGetSleepTxt">
                 <property name="text">
                  <string>go to sleep</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="BtnSave">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder></zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
